Dockerfile.DS

```diff
@@ -9,21 +9,25 @@ RUN set -x \
     # install python and dependencies
     && apk update \
     && apk --no-cache \
-        --repository http://dl-4.alpinelinux.org/alpine/v3.7/community \
-        --repository http://dl-4.alpinelinux.org/alpine/v3.7/main \
+        --repository http://dl-4.alpinelinux.org/alpine/v3.14/community \
+        --repository http://dl-4.alpinelinux.org/alpine/v3.14/main \
         --arch=x86_64 add \
         python3 \
         su-exec \
         gcc \
         g++ \
         git \
-        py3-zmq \
+        py3-pyzmq \
         pkgconfig \
         zeromq-dev \
         musl-dev \
         mercurial \
-    && pip3 install --upgrade pip==9.0.3 \
-    && cp /usr/bin/python3.6 /usr/bin/python \
+        py3-pip \
+        libffi-dev \
+        python3-dev \
+        mesa-dev \
+    && pip3 install --upgrade pip==21.3.1 \
+    && cp /usr/bin/python3.9 /usr/bin/python \
     ## install Go
     && apk --update-cache --allow-untrusted \
         --repository http://dl-4.alpinelinux.org/alpine/edge/community \
```

Comment:
I don't think this is needed anymore (with >=3.14).
ditto for the `--allow-untrusted` argument above.